app/controllers/home_controller.rb

```diff
@@ -27,12 +27,15 @@ def contact_us
 	end
 
 	def send_contact_form
-		@contact_message = ContactMessage.new(message_params)
-		if @contact_message.valid? && verify_recaptcha
+		@contact_message = ContactMessage.new(message_params)	  
+		success = verify_recaptcha(action: 'contact_us', minimum_score: 0.8, secret_key: Rails.application.secrets.reCaptcha_secret)
```

Comment:
I tested defining the score to 1 and it only shows a toast message indicating the recaptcha validation failed and the form fields cleared